plugins/outputs/cloudwatchlogs/internal/pusher/target.go

```diff
@@ -216,6 +233,8 @@ func (m *targetManager) calculateBackoff(attempt int) time.Duration {
 		delay = maxDelay
 	}
 
-	jitter := time.Duration(float64(delay) * (float64(time.Now().UnixNano()%100) / 500.0 * jitterFactor))
+	//nolint:gosec // using math/rand is acceptable for jitter calculation
+	r := rand.New(rand.NewSource(time.Now().UnixNano()))
```

Comment:
Can reuse random generator from the retry.
https://github.com/aws/amazon-cloudwatch-agent/blob/799f0c3bdcfc18fcadf3b5e0c1c9ebaa543e18a8/plugins/outputs/cloudwatchlogs/internal/pusher/retry.go#L38